roles/devscripts/vars/main.yml

```diff
@@ -33,6 +33,7 @@ cifmw_devscripts_rhsm_enabled_repos: false
 
 cifmw_devscripts_config_defaults:
   working_dir: "/home/dev-scripts"
+  assets_extra_folder: "/home/dev-scripts/assets"
```

Comment:
a pity dev-scripts doesn't have converging name to either `*_dir` or `*_folder` - we'd be able to just extract those in a loop...